docs/otel-with-elastic.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[open-telemetry-with-elastic]]
+=== OpenTelemetry API/SDK with Elastic APM agents
+
+++++
+<titleabbrev>OpenTelemetry with Elastic APM agents</titleabbrev>
+++++
+
+The Elastic OpenTelemetry bridge translates calls to the OpenTelemetry API to Elastic APM API calls.
+This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans.
+
+TIP: If you'd like to use OpenTelemetry to send data directly to the APM server instead,
+see <<open-telemetry-direct>>.
+
+See the relevant Elastic APM agent documentation to get started:
```

Comment:
You mentioned .NET is also supported but I can't find documentation for that agent.

Reply:
@gregkalapos will follow-up on this. There's no documentation now, though there is a bridge available.

Reply:
Opened doc PR for .NET: https://github.com/elastic/apm-agent-dotnet/pull/1966

Reply:
Thanks! I'll add .NET to this list.